src/node_options.cc

```diff
@@ -449,6 +449,8 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             "write warnings to file instead of stderr",
             &EnvironmentOptions::redirect_warnings,
             kAllowedInEnvironment);
+  AddOption("--test-udp-no-try-send", "",  // For testing only.
```

Comment:
Does having a zero-length help text make the option hidden from the `--help` options list? If not, maybe that would be a good idea for this?

Reply:
Yes, it hides the option.